src/core.c/Instant.rakumod

```diff
@@ -4,8 +4,8 @@ my class Duration {... }
 
 my class Instant is Cool does Real {
     has Int $.tai is default(0);
-      # A linear count of nanoseconds since 1970-01-01T00:00:00Z, plus
-      # Rakudo::Internals.initial-offset. Thus, $.tai matches TAI from 1970
+      # A count of nanoseconds since 1970-01-01T00:00:00Z with leap seconds
+      # and Rakudo::Internals.initial-offset. Thus, $.tai matches UTC from 1970
```

Comment:
Why the s/TAI/UTC ??

Reply:
Some sites equate TAI as equivalent to UTC, but  TAI  is not corrected from 86400 seconds per day.

Reply:
Which is why TAI is mentioned there, instead of UTC?